narwhals/_arrow/group_by.py

```diff
@@ -20,33 +20,28 @@
     from narwhals._arrow.expr import ArrowExpr
     from narwhals._arrow.typing import IntoArrowExpr
 
-POLARS_TO_ARROW_AGGREGATIONS = {
-    "sum": "sum",
-    "mean": "mean",
-    "median": "approximate_median",
-    "max": "max",
-    "min": "min",
-    "std": "stddev",
-    "var": "variance",  # currently unused, we don't have `var` yet
-    "len": "count",
-    "n_unique": "count_distinct",
-    "count": "count",
-}
-
-
-def get_function_name_option(
-    function_name: str,
-) -> pc.CountOptions | pc.VarianceOptions | None:
-    """Map specific pyarrow compute function to respective option to match polars behaviour."""
+
+def polars_to_arrow_aggregations() -> (
+    dict[str, tuple[str, pc.VarianceOptions | pc.CountOptions | None]]
+):
+    """Map polars compute functions to their pyarrow counterparts and options that help match polars behaviour."""
     import pyarrow.compute as pc
 
-    function_name_to_options = {
-        "count": pc.CountOptions(mode="only_valid"),
-        "count_distinct": pc.CountOptions(mode="all"),
-        "stddev": pc.VarianceOptions(ddof=1),
-        "variance": pc.VarianceOptions(ddof=1),
+    return {
+        "sum": ("sum", None),
+        "mean": ("mean", None),
+        "median": ("approximate_median", None),
+        "max": ("max", None),
+        "min": ("min", None),
+        "std": ("stddev", pc.VarianceOptions(ddof=1)),
+        "var": (
+            "variance",
+            pc.VarianceOptions(ddof=1),
+        ),  # currently unused, we don't have `var` yet
+        "len": ("count", pc.CountOptions(mode="all")),
+        "n_unique": ("count_distinct", pc.CountOptions(mode="all")),
+        "count": ("count", pc.CountOptions(mode="only_valid")),
     }
```

Comment:
i might be missing something but why not make this dict a constant at the top of the file instead of putting it in a function?

Reply:
It was to avoid importing `pc` at the top of the file; maybe is there a simpler way?

Reply:
ah i see - tbh i think it's ok to import it at the top, we use pc all over the place in _arrow anyway

Reply:
ah nevermind, it would fail the pre-commit checks. this is fine then, ignre me